pages/common/omf.md

```diff
@@ -0,0 +1,28 @@
+# omf
+
+> Oh My Fish, the Fishshell Framework.
+> Install packages to extend and modify the fish shell.
+
+- Install one or more packages:
+
+`omf install {{name}}`
+
+- List installed packages:
+
+`omf list`
+
+- List available themes:
+
+`omf theme`
+
+- Apply a theme:
+
+`omf theme {{name}}`
+
+- Remove a theme or package:
```

Comment:
What if there is a theme and package with the same name ? Both will be removed ?

Reply:
Hm themes are also packages, so it doesn't matter.

Reply:
I see. Ok.